### ISSUE TITLE
Linkedin login form- noscrolling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manieyedone85/socialauth-android#94]
```
What steps will reproduce the problem?
1.logging into linkedin
2.linkedin form appears with no scroll
3.

What is the expected output? What do you see instead?
when trying to login to linkedin the form that appears does not have a scroll 
bar which makes logging more difficult.

What version of the product are you using? On what operating system?
2.5, windows 7, eclipse helios

Please provide any additional information below.
mobile phone used: LG p500h. 3.2 inches, 2.2 android version.
```

Original issue reported on code.google.com by `frimaw...@gmail.com` on 20 Jun 2013 at 3:30

**Comment by GoogleCodeExporter** [comment]
```
can you please let me know screen resolution of ur device.
```

Original comment by `vineet.a...@3pillarglobal.com` on 21 Jun 2013 at 5:54

**Comment by GoogleCodeExporter** [comment]
```
Hi,

screen resolution: 320 x 480 (~180dpi)
```

Original comment by `frimaw...@gmail.com` on 21 Jun 2013 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
Thanks , will check and let you know soon
```

Original comment by `vineet.a...@3pillarglobal.com` on 24 Jun 2013 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
Hi , the issue is solved and will come in latest release 2.6 schedule on Monday.
```

Original comment by `l...@3pillarglobal.com` on 27 Jun 2013 at 10:58
- Changed state: **Fixed**
